feature/experimental/p4rt/otg_tests/google_discovery_protocol_packetout_lag_test/google_discovery_protocol_packetout_lag_test.go

```diff
@@ -0,0 +1,593 @@
+		s4.Enabled = ygot.Bool(true)
+	}
+	s4a := s4.GetOrCreateAddress(a.IPv4)
+	s4a.PrefixLength = ygot.Uint8(ipv4PLen)
+
+	if hasVlan && deviations.P4RTGdpRequiresDot1QSubinterface(dut) {
+		s1 := i.GetOrCreateSubinterface(1)
+		s1.GetOrCreateVlan().GetOrCreateMatch().GetOrCreateSingleTagged().SetVlanId(vlanID)
+		if deviations.NoMixOfTaggedAndUntaggedSubinterfaces(dut) {
+			s.GetOrCreateVlan().GetOrCreateMatch().GetOrCreateSingleTagged().SetVlanId(10)
+			i.GetOrCreateAggregation().GetOrCreateSwitchedVlan().SetNativeVlan(10)
+		}
+	}
+
+	return i
+}*/
+
+// configureDUT configures agg1 and agg2 on the DUT.
+func configureDUT(t *testing.T, dut *ondatra.DUTDevice) []string {
+	t.Helper()
+	fptest.ConfigureDefaultNetworkInstance(t, dut)
+	var aggIDs []string
+	for aggIdx, a := range []*aggPortData{agg1, agg2} {
+		b := &gnmi.SetBatch{}
+		d := &oc.Root{}
+
+		aggID := netutil.NextAggregateInterface(t, dut)
+		aggIDs = append(aggIDs, aggID)
+
+		agg := d.GetOrCreateInterface(aggID)
+		agg.GetOrCreateAggregation().LagType = oc.IfAggregate_AggregationType_STATIC
+		agg.Type = oc.IETFInterfaces_InterfaceType_ieee8023adLag
+		agg.Description = ygot.String(a.ateAggName)
+		if deviations.InterfaceEnabled(dut) {
+			agg.Enabled = ygot.Bool(true)
+		}
+		s := agg.GetOrCreateSubinterface(0)
+		s4 := s.GetOrCreateIpv4()
+		if deviations.InterfaceEnabled(dut) {
+			s4.Enabled = ygot.Bool(true)
+		}
+		a4 := s4.GetOrCreateAddress(a.dutIPv4)
+		a4.PrefixLength = ygot.Uint8(ipv4PLen)
+
+		gnmi.BatchDelete(b, gnmi.OC().Interface(aggID).Aggregation().MinLinks().Config())
+		gnmi.BatchReplace(b, gnmi.OC().Interface(aggID).Config(), agg)
+
+		p1 := dut.Port(t, fmt.Sprintf("port%d", (aggIdx*1)+1))
+		// p2 := dut.Port(t, fmt.Sprintf("port%d", (aggIdx*2)+2))
+		for _, port := range []*ondatra.Port{p1} {
+			gnmi.BatchDelete(b, gnmi.OC().Interface(port.Name()).Ethernet().AggregateId().Config())
+			i := d.GetOrCreateInterface(port.Name())
+			// i := &oc.Interface{Name: ygot.String(p1.Name()), Id: ygot.Uint32(a.aggPortID)}
+			// i := d.Interface{Name: ygot.String(port.Name()), Id: ygot.Uint32(a.aggPortID)}
+			i.Id = ygot.Uint32(a.aggPortID)
+			i.Description = ygot.String(fmt.Sprintf("LAG - Member -%s", port.Name()))
+			e := i.GetOrCreateEthernet()
+			e.AggregateId = ygot.String(aggID)
+			i.Type = oc.IETFInterfaces_InterfaceType_ethernetCsmacd
+			if a.hasVlan && deviations.P4RTGdpRequiresDot1QSubinterface(dut) {
+				s1 := i.GetOrCreateSubinterface(1)
+				s1.GetOrCreateVlan().GetOrCreateMatch().GetOrCreateSingleTagged().SetVlanId(vlanID)
+				if deviations.NoMixOfTaggedAndUntaggedSubinterfaces(dut) {
+					s.GetOrCreateVlan().GetOrCreateMatch().GetOrCreateSingleTagged().SetVlanId(10)
+					i.GetOrCreateAggregation().GetOrCreateSwitchedVlan().SetNativeVlan(10)
+				}
+			}
+			if deviations.InterfaceEnabled(dut) {
+				i.Enabled = ygot.Bool(true)
+			}
+			gnmi.BatchReplace(b, gnmi.OC().Interface(port.Name()).Config(), i)
+		}
+
+		b.Set(t, dut)
+	}
```

Comment:
Would require below deviation code for `ExplicitInterfaceInDefaultVRF` here before LAG interface status is checked - 

```golang
       if deviations.ExplicitInterfaceInDefaultVRF(dut) {
               for _, aggID := range aggIDs {
                       fptest.AssignToNetworkInstance(t, dut, aggID, deviations.DefaultNetworkInstance(dut), 0)
               }
       }
```